scripts/src/java/org/oppia/android/scripts/label/AccessibilityLabelCheck.kt

```diff
@@ -0,0 +1,156 @@
+package org.oppia.android.scripts.label
+
+import org.oppia.android.scripts.proto.AccessibilityLabelExemptions
+import org.w3c.dom.Node
+import org.w3c.dom.NodeList
+import java.io.File
+import java.io.FileInputStream
+import java.util.stream.Collectors
+import java.util.stream.IntStream
+import javax.xml.parsers.DocumentBuilderFactory
+
+/**
+ * Script for ensuring that all the Activities in the codebase are defined with accessibility
+ * labels.
+ *
+ * Usage:
+ *   bazel run //scripts:accessibility_label_check -- <path_to_directory_root> <path_to_app_level
+ *   _manifest_file>
+ *
+ * Arguments:
+ * - path_to_directory_root: directory path to the root of the Oppia Android repository.
+ * - path_to_app_level_manifest_file: path to the manifest file of app layer.
```

Comment:
Rather than 'app-level manifest', I suggest framing it as the 'main manifest of the app' since there won't be an "app layer" or "app module" in the future.

That being said, I think we should actually make this generic & allow any number of activities to be passed in (& test multiple in the tests). Longer term, we'll be moving away from a single activity to multiple that are merged together using the Bazel Android manifest merging. This actually lets us properly exclude test activities from the production build that we ship to users. Rather than needing to update this check then, it'd be great if we could just support it in the initial version. Do you have any concerns with this?

Reply:
Revamped the approach to make it generic, now we can pass multiple manifests.